quarkus/config-api/src/main/java/org/keycloak/config/DatabaseOptions.java

```diff
@@ -106,6 +106,11 @@ public class DatabaseOptions {
             .description("Deactivate specific named datasource <datasource>.")
             .build();
 
+    public static final Option<String> DB_POSTGRESQL_TARGET_SERVER_TYPE = new OptionBuilder<>("db-jdbc-default-options", String.class)
```

Comment:
Should we make this specific to the `targetServerType`?
```suggestion
    public static final Option<String> DB_POSTGRESQL_TARGET_SERVER_TYPE = new OptionBuilder<>("db-postgres-target-server-type", String.class)
```